packages/markdown/src/index.tsx

```diff
@@ -1,6 +1,5 @@
 import './codemirrorImports';
 
 export { MarkdownEditorConnected as MarkdownEditor } from './MarkdownEditor';
-export { MarkdownPreview } from './components/MarkdownPreview';
```

Comment:
Why?

Reply:
This was an attempt to remove it from the main bundle when lazy loading however it didn't work, want me to revert?

Reply:
It's a breaking change technically so let's revert it and revisit in the follow up

Reply:
done!